test/tests/interfacekernels/1d_interface/ADMatreaction_1D_steady.i

```diff
@@ -0,0 +1,172 @@
+# Steady-state test for the ADMatInterfaceReaction kernel.
+#
+# Specie M transport from domain 1 (0<=x<=1) to domain 2 (1<x<=2),
+# u and v are concentrations in domain 1 and domain 2.
+#
+# Diffusion in both domains can be described by Ficks law and diffusion
+# kernel is applied.
+#
+# Specie M has different diffusity in different domains, here set as D1=4, D2=2.
+#
+# Dirichlet boundary conditions are applied, i.e., u(0)=1, v(2)=0
+#
+# At the interface consider the following
+#
+# (a) Fluxes are matched from both domains (InterfaceDiffusion kernel)
+#
+# (b) First-order reaction is R = kf*u - kb*v
+#
+# Analytical solution is
+# u = -0.2*u+1,    0<=u<=1
+# v = -0.4*v+0.8,  1<v<=2
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 1
+    nx = 10
+    xmax = 2
+  []
+  [./subdomain1]
+    input = gen
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '1.0 0 0'
+    block_id = 1
+    top_right = '2.0 1.0 0'
+  [../]
```

Comment:
Remove all `./` and `../` syntax from the input as they're no longer needed by the parser